examples/image_classification-tf.ipynb

```diff
@@ -0,0 +1,7883 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #78.        image = tf.image.crop_to_bounding_box(
</pre><p>How about following the standard transformation chain for validation images? </p><p> </p><p>Resizing (256 x 256) -&gt; Central Cropping (with an area factor of 0.875 which amounts to 224 x 224)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>